mappings/net/minecraft/server/network/ServerPlayerEntity.mapping

```diff
@@ -33,13 +34,15 @@ CLASS net/minecraft/class_1637 net/minecraft/server/network/ServerPlayerEntity
 		ARG 1 distance
 	METHOD method_6066 updateBlockEntity (Lnet/minecraft/class_348;)V
 		ARG 1 blockEntity
-	METHOD method_6069 openContainer (Lnet/minecraft/class_1983;)V
+	METHOD method_6069 registerScreenHandler (Lnet/minecraft/class_1983;)V
```

Comment:
not open?

Reply:
yep. 
it actually sends packets to the client with screen handler info

Reply:
it doesn't register it right

Reply:
wait it doesnt

Reply:
fixed

Reply:
`updateScreenHandler`